src/deps/tinycthread.h

```diff
@@ -23,7 +23,7 @@ freely, subject to the following restrictions:
 
 #ifndef _TINYCTHREAD_H_
 #define _TINYCTHREAD_H_
-
+#if defined(STDC_NO_THREADS)
```

Comment:
tinycthread.h inclusion is already guarded by a macro, so no need to add an additional #if in it.